robot_framework/sap/ryk_afklar.py

```diff
@@ -0,0 +1,230 @@
+
+    # Delete "Maks antal træffere"
+    session.findById("wnd[0]/usr/txt%%DYN012-LOW").text = ""
+
+    # Search
+    session.findById("wnd[0]/tbar[1]/btn[8]").press()
+
+    # Apply layout
+    session.findById("wnd[0]/tbar[1]/btn[33]").press()
+    layout_table = session.findById("wnd[1]/usr/ssubD0500_SUBSCREEN:SAPLSLVC_DIALOG:0501/cntlG51_CONTAINER/shellcont/shell")
+    layout_index = gridview_util.find_row_index_by_value(layout_table, "TEXT", "RPA layout til håndtering af ryk_afklar")
+    layout_table.setCurrentCell(layout_index, "TEXT")
+    layout_table.clickCurrentCell()
+
+    # Find number of cases
+    case_table = session.findById("wnd[0]/usr/cntlGRID1/shellcont/shell")
+    return case_table.rowCount
+
+
+def handle_case(orchestrator_connection: OrchestratorConnection, session, row_index: int):
+    """Open a case and try to add the bilag to an existing fp-aftale.
+
+    Args:
+        orchestrator_connection: The connection to OpenOrchestrator.
+        session: The SAP session object.
+        row_index: The row index of the case to handle in the work list.
+
+    Raises:
+        ValueError: If the bilag wasn't found in the forretningspartner's postliste.
+    """
+    case_table = session.findById("wnd[0]/usr/cntlGRID1/shellcont/shell")
+
+    # Scroll to row
+    case_table.firstVisibleRow = row_index
+
+    # Get the forretningspartner and bilagsnummer
+    forretningspartner = case_table.getCellValue(row_index, "GPART")
+    bilagsnummer = case_table.getCellValue(row_index, "OPBEL")
+
+    reference = f"{forretningspartner}:{bilagsnummer}"
+    if not check_queue(orchestrator_connection, reference):
+        return
+
+    queue_element = orchestrator_connection.create_queue_element(config.QUEUE_NAME, reference)
+    orchestrator_connection.set_queue_element_status(queue_element.id, QueueStatus.IN_PROGRESS)
+
+    # Open forretningspartneroversigt
+    case_table.selectedRows = row_index
+    session.findById("wnd[0]/tbar[1]/btn[18]").press()
+    session.findById("wnd[1]/usr/subSUB1:SAPLZDKD_DCL_CLARIFICATION:0100/btnBUTTON_FMCACOV").press()
+
+    # Find the bilag in the postliste table and click "Vis relation"
+    postliste_table = session.findById("wnd[0]/usr/tabsDATA_DISP/tabpDATA_DISP_FC1/ssubDATA_DISP_SCA:RFMCA_COV:0202/cntlRFMCA_COV_0100_CONT5/shellcont/shell")
+
+    bilag_row = gridview_util.find_row_index_by_value(postliste_table, "OPBEL", bilagsnummer)
+
+    if bilag_row == -1:
+        raise ValueError("The bilagsnummer wasn't found.")
+
+    postliste_table.setCurrentCell(bilag_row, "OPBEL")
+    postliste_table.contextMenu()
+    postliste_table.selectContextMenuItemByText("Vis relation")
+
+    # Find the fp-aftale if any
+    aftale_row = find_fp_aftale(postliste_table, bilagsnummer)
+
+    # If a fp-aftale was found add the bilag to it
+    if aftale_row != -1:
+        aftale_added = add_bilag_to_aftale(session, aftale_row, bilagsnummer)
+    else:
+        aftale_added = False
```

Comment:
```suggestion
    aftale_added = add_bilag_to_aftale(session, aftale_row, bilagsnummer) if aftale_row != -1 else False
```

Reply:
I really think the former is more readable even though it might be "incorrect"